pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -492,7 +492,18 @@ def _set_parallel_devices_and_init_accelerator(self) -> None:
                     "The flag `devices` must be an int with `accelerator='cpu'`,"
                     f" got `devices={self._devices_flag}` instead."
                 )
-
+        if not self.accelerator.is_available():
+            if isinstance(self.accelerator, GPUAccelerator):
+                hardware_str = "GPU"
+            elif isinstance(self.accelerator, IPUAccelerator):
+                hardware_str = "IPU"
+            else:
+                hardware_str = "TPU"
```

Comment:
To match `strategy_name`

```suggestion
                hardware_name = "TPU"
```